### DISCUSSION TITLE
Apparent memory leak, web UI unresponsive

### DISCUSSION THREAD

**Discussion by tomhalley** [haugene/docker-transmission-openvpn#2807] [Q&A]
I'm running this image on a server with about 4gb of allocated memory, and I'm constantly running into issues where the used memory is 100% and the web UI becomes unresponsive. The container will often also be unresponsive when trying to restart/recreate through portainer.

Originally the host machine had 4gb of total memory but I upgraded to 8gb to alleviate this apparent issue. The problem only seems to be when downloading files over 10gb in size. 

Should I expect to have this issue every time I download something over 4gb in size?

**Comment by pkishino** [comment]
What underlying platform are you running this on?
I used to run this on a Mac mini but have moved it to my asustor NAS with 4gb ram and no such issues、sometimes it responds a bit slower because I run a few other containers on it as well but usage of the transmission one isn’t so high.

**Reply by tomhalley** [reply]
Underlying is Ubuntu 22.04.3 LTS

**Reply by pkishino** [reply]
Raise an issue on official transmission repo then please.
I know they fixed won’ some big memory issues when they moved to 4.x etc so make sure you actually are running latest release of this container which uses 4.0.5(latest official release) of the transmission client
